demo/app/lib/wallet_sdk/wallet_sdk_js.dart

```diff
@@ -177,6 +177,10 @@ class WalletSDK extends WalletPlatform {
     }
   }
 
+  Future<WalletSDKError> parseWalletSDKError({required String localizedErrorMessage}) async {
+    return WalletSDKError.fromJson({});
```

Comment:
I think it is better to throw an error, something like `throw Exception('Method not implemented')`